src/buildfromsource/FSharp.Compiler.Private/FSComp.fs

```diff
@@ -4309,6 +4309,12 @@ type internal SR private() =
     /// The file '%s' changed on disk unexpectedly, please reload.
     /// (Originally from ..\FSComp.txt:1427)
     static member ilreadFileChanged(a0 : System.String) = (3223, GetStringFunc("ilreadFileChanged",",,,%s,,,") a0)
+    /// '%s' cannot implement the interface '%s' with the two instantiations '%s' and '%s' because they may unify.
+    /// (Originally from ..\FSComp.txt:1427)
+    static member typrelInterfaceWithConcreteAndVariable(a0 : System.String, a1 : System.String, a2 : System.String, a3 : System.String) = (3223, GetStringFunc("typrelInterfaceWithConcreteAndVariable",",,,%s,,,%s,,,%s,,,%s,,,") a0 a1 a2 a3)
+    /// You cannot implement the interface '%s' with the two instantiations '%s' and '%s' because they may unify.
+    /// (Originally from ..\FSComp.txt:1428)
+    static member typrelInterfaceWithConcreteAndVariableObjectExpression(a0 : System.String, a1 : System.String, a2 : System.String) = (3223, GetStringFunc("typrelInterfaceWithConcreteAndVariableObjectExpression",",,,%s,,,%s,,,%s,,,") a0 a1 a2)
```

Comment:
Why two messages? The first is for object expressions, where we don't have a class name. The second is for classes, where I want to include the class name.

Reply:
That makes sense to me.